### PR TITLE
Setting Up Hello Microverse project

### PR DESCRIPTION
**Pull request by Haywayaheadshot**
## What?
I created HTML, CSS, and test.md files for the project. I added linters to check for bugs.

## Why?
These were required tasks by Microverse. For me to move to the stage of my building on my software development career.

## How?
I wrote the code for HTML and CSS so it would read "Hello Microverse!". I created a folder in which a file named "workflows" was created. I then went ahead and paste the linters code ".github/workflows/linters.yml" in it.

## Anything Else?
I added test.md to .gitignore list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (codecaiine) [inline README.md:62]: - [ ] The link to your license is broken :cry: , please fix it in order to make your readme more professional :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Haywayaheadshot , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Project is professional :heavy_check_mark: 
- Good use of Github flow :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Great commits messages :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please edit your Pull Request to make it a more descriptive t, it should tell what changes are inside to the reviewer . A good example will be **Set up a Hello Microverse project** instead of  **First project feature**  :+1: 
- [ ] Please edit also your Pull Request summary because like your PR title the summary should describe introduced changes in more detail. :+1: 
You can check the example below :arrow_down: 
Screenshot from 2022-08-16 18-25-08

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Haywayaheadshot) [reply README.md]: Thank you, Yannick. I have resubmitted the link with the necessary corrections.

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](HelloWorld/LICENSE.txt) licensed.
```
Comment (ichala) [inline README.md:62]: - [ ] The link to your license is still broken, as the previous reviewer requested, please try to fix it , I suggest changing `[MIT](HelloWorld/LICENSE.txt)` to `[LICENSE](./LICENSE.txt)`

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Haywayaheadshot , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


the-simpsons-homer-simpson

# To Highlights

- Project is professional ✔️
- Good use of Github flow ✔️
- Great commits messages ✔️
- PR title and description now look professional ✔️

-----------------

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
N/A
-----------------------------


***********************************
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @ichala in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Haywayaheadshot) [reply README.md]: Hi Ichala. Thank you. I have rewritten it in accordance with your suggestion and it is fixed.

**Review by levy002 — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @Haywayaheadshot  👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💪🏻

✅ You used the correct Github Flow.
✅ Professional README.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

  - [OPTIONAL] Kindly make sure to write your commit message in imperative tense and capitalized. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**